app/components/date-time-format-filter/date-time-format-filter.js

```diff
@@ -3,22 +3,30 @@
 angular.module('bulbs.cms.dateTimeFilter', [
   'bulbs.cms.site.config',
   'lodash',
-  'moment'
+  'moment',
 ])
   .filter('dateTimeFormat', [
     '_', 'moment', 'CmsConfig',
     function (_, moment, CmsConfig) {
-      return function (date, format) {
+      function isInvalidDateValue (dateValue) {
+        return !(
```

Comment:
Too much analysis is required to understand this statement, distribute the not operator so this is more readable.

```
return !_.isString(dateValue) && !_.isDate(dateValue) && !moment.isMoment(dateValue);
```

Or make this a positive function `isValidDateValue` and do `!isValidDateValue` when doing the check.

Also is `isInvalidDateValue('my string value') === false`? If so, this check should be more specific about what a valid `String` is.

Checking the moment docs, you might be able to just use [`moment(<date-like-thing>).isValid()`](http://momentjs.com/docs/#/parsing/is-valid/) or [`moment.invalid(<date-like-thing>)`](http://momentjs.com/docs/#/utilities/invalid/).